app/Detail_Album.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Detail_Album extends Model
```

Comment:
Tên model dựa theo tên bảng sẽ là DetailAlbum nhé
như vậy thì Laravel mới tìm bảng detail_album
viết như em sẽ nhầm thành bảng detail__album
https://laravel.io/forum/06-26-2014-naming-conventions-for-model-classes